### ISSUE TITLE
Sysmon 11.10 stops capturing network events after a period of time

### ISSUE THREAD

**Issue by puQy** [MicrosoftDocs/sysinternals#295]
**Issue Description:**

After updating Sysmon  10.42 to 11.10, I have unfortunately recognized, that the Sysmon service stops capturing network events (Event ID: 3) after a period of time of round about 45 minutes (sometimes less than 45 minutes). To fix the issue I have tried to re-install the sysmon service and also to update the configuration file. However, the service stops capturing network events after a short period of time.  Furthermore, to verify that the issue is not on my system only, my fellows and I checked also their Sysmon logs and noticed the same behavior.

We also have checked if the network monitoring is enabled and it is.

```
Current configuration:
 - Service name:                  Sysmon64
 - Driver name:                   SysmonDrv

 - HashingAlgorithms:             SHA1,MD5,SHA256,IMPHASH
 - Network connection:            enabled
 - Image loading:                 enabled
 - CRL checking:                  enabled
 - DNS lookup:                    disabled
 - Filter archive directory:      \Sysmon\
```

**System:**
Windows 10.0.18362.904
Sysmon 11.10

**Comment by analyze-v** [comment]
Hello
could you contact me offline at syssite@microsoft.com and I will try to assist you with this.

MarkC(MSFT)

**Comment by super0xbad1dea** [comment]
I have the same issue. I've seen this on Windows Server 2016 10.0.14393

**Comment by analyze-v** [comment]
Hello
I am looking at this today. Would you happen to have a memory dump from an affected endpoint that you could share with us? If so could you contact me at marcook@microsoft.com<mailto:marcook@microsoft.com> and I will provide you with an upload location.

Regards

Mark

**Comment by lukekim** [comment]
Resolved in Sysmon 11.11, closing.
